core/app/assets/javascripts/refinery/admin.js.erb

```diff
@@ -577,13 +577,17 @@ var page_options = {
   },
 
   page_part_dialog: function(){
+    var slugOptions = {
+      separator: '_'
```

Comment:
Sorry, i'm not agreed. 
We don't speak about the page slug but the page part slug.

The slug for the page part is never displayed as an entire page so Google will never see it.
 
When we call a page part in an html view, we write something like that : 
`<%= raw @page.content_for(:side_body) %>`
not
`<%= raw @page.content_for(:"side-body") %>`